modules/ipi-install-configuring-host-dual-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -0,0 +1,136 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-host-dual-network-interfaces-in-the-install-config-yaml-file_{context}"]
+= Optional: Configuring host network interfaces for dual NIC
+
+Before installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState to support dual NIC.
+
+.Prequisites
+
+* Configure a `PTR` DNS record with a valid hostname for each node with a static IP address.
+* Install the NMState CLI (`nmstate`).
+
+.Procedure
+
+. Use the `networkConfig` configuration setting by adding the NMState configuration to hosts within the `install-config.yaml` file:
++
+[source,yaml]
+----
+hosts:
+ - hostname: worker-1
+    interfaces:
+     - name: eno1
+       macAddress: 0c:42:a1:55:f3:06
+     - name: eno2
+       macAddress: 0c:42:a1:55:f3:07
+    networkConfig: <1>
+      interfaces: <2>
```

Comment:
Apply the same suggestions as mentioned in `modules/agent-install-sample-config-bond-sriov.adoc`